presto-benchto-benchmarks/src/test/resources/sql/presto/tpcds/q45.plan.txt

```diff
@@ -4,27 +4,26 @@ local exchange (GATHER, SINGLE, [])
-                        semijoin (PARTITIONED):
-                            remote exchange (REPARTITION, HASH, ["i_item_id"])
-                                join (INNER, REPLICATED):
-                                    join (INNER, PARTITIONED):
-                                        remote exchange (REPARTITION, HASH, ["ws_bill_customer_sk"])
-                                            join (INNER, REPLICATED):
-                                                scan web_sales
-                                                local exchange (GATHER, SINGLE, [])
-                                                    remote exchange (REPLICATE, BROADCAST, [])
-                                                        scan date_dim
-                                        local exchange (GATHER, SINGLE, [])
-                                            remote exchange (REPARTITION, HASH, ["c_customer_sk"])
-                                                join (INNER, PARTITIONED):
-                                                    remote exchange (REPARTITION, HASH, ["c_current_addr_sk"])
-                                                        scan customer
-                                                    local exchange (GATHER, SINGLE, [])
-                                                        remote exchange (REPARTITION, HASH, ["ca_address_sk"])
-                                                            scan customer_address
+                        semijoin (REPLICATED):
+                            join (INNER, REPLICATED):
+                                join (INNER, PARTITIONED):
+                                    remote exchange (REPARTITION, HASH, ["ws_bill_customer_sk"])
+                                        join (INNER, REPLICATED):
+                                            scan web_sales
+                                            local exchange (GATHER, SINGLE, [])
+                                                remote exchange (REPLICATE, BROADCAST, [])
+                                                    scan date_dim
                                     local exchange (GATHER, SINGLE, [])
-                                        remote exchange (REPLICATE, BROADCAST, [])
-                                            scan item
+                                        remote exchange (REPARTITION, HASH, ["c_customer_sk"])
+                                            join (INNER, PARTITIONED):
+                                                remote exchange (REPARTITION, HASH, ["c_current_addr_sk"])
+                                                    scan customer
+                                                local exchange (GATHER, SINGLE, [])
+                                                    remote exchange (REPARTITION, HASH, ["ca_address_sk"])
+                                                        scan customer_address
+                                local exchange (GATHER, SINGLE, [])
```

Comment:
did you verify these results?

Reply:
@findepi?

Reply:
https://github.com/prestodb/presto/pull/11957#issuecomment-448229964 ?